func-complete-check/lib/to-bool.ts

```diff
@@ -0,0 +1,9 @@
+export function primitiveToBoolean (value: string | number | boolean | null | undefined): boolean {
+  if (value === 'true') {
```

Comment:
@GuyHarwood why not using the library `To-Bool`?

Reply:
because it is not supported by typescript, nor does it have a `@types` file.  it behaves differently with typescript.  i spent a while working on it.